CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## unreleased
 * `parse_regex_all` `pattern` param  can now be resolved from a variable
 * fixed `parse_json` data corruption issue for numbers greater or equal to `i64::MAX` 
+* support timestamp comparison using operators <, <=, >, >=
```

Comment:
💭 Can you add an example here as well: https://vector.dev/docs/reference/vrl/expressions/#comparison-examples

Reply:
Sure, will try to remember to do it a bit later.